core-bundle/assets/styles/template-studio/common.pcss

```diff
@@ -115,15 +113,16 @@
     }
 
     .shadowed {
-        background-image: linear-gradient(
-            45deg,
-            var(--panel-bg) 25%,
-            transparent 25%,
-            transparent 50%,
-            var(--panel-bg) 50%,
-            var(--panel-bg) 75%,
-            transparent 75%
-        );
+        background-image:
+            linear-gradient(
+                45deg,
+                var(--panel-bg) 25%,
+                transparent 25%,
+                transparent 50%,
+                var(--panel-bg) 50%,
+                var(--panel-bg) 75%,
+                transparent 75%
+            );
```

Comment:
This is worse than before now. Why is `never-multi-line` not applied here?

Reply:
The rule in place is actually `@stylistic/declaration-colon-newline-after`, I agree that this is worse but we have to disable the rule completely:
https://github.com/stylelint-stylistic/stylelint-stylistic/tree/main/lib/rules/declaration-colon-newline-after

See 9e5f27c5c3a8f7e9023faa8b467a83a5568b6b71